en/components/tree.md

```diff
@@ -0,0 +1,313 @@
+```html
+<igx-tree [selection]="'BiState'">
+</igx-tree>
+```
+### Cascading
+To enable cascade node selection in the `igx-tree` just set the selection property to **Cascading**. This will render a checkbox for every node. 
+```html
+<igx-tree [selection]="'Cascading'">
+</igx-tree>
+```
+In this mode a parent's selection state entirely depends on the selection state of its children. When a parent has some selected and some deselected children, its checkbox is in an indeterminate state.
+
+### Angular Tree Checkbox
+The Angular Tree component provides built-in support for checkboxes, allowing users to select more than one item.
+
+The TreeView checkboxes also have a tri-state mode, which is applicable only for parent nodes. In this mode, the parent node will go into the indeterminate state when the child nodes are partially checked.
+## Keyboard Navigation
+Keyboard navigation in IgxTree provides a rich variety of keyboard interactions for the user. This functionality is enabled by default and allows users to navigate through the nodes.
+
+The IgxTree navigation is compliant with W3C accesibility standards and convenient to use.
+
+**Key Combinations**
+
+ - <kbd>Arrow Down</kbd> - navigates to the next visible node. Marks the node as active. Does nothing if on the LAST node
+ - <kbd>Ctrl + Arrow Down</kbd> - navigates to the next visible node. Does nothing if on the LAST node
+ - <kbd>Arrow Up</kbd> - navigates to the previous visible node. Marks the node as active. Does nothing if on the FIRST node
+ - <kbd>Ctrl + Arrow Up</kbd> - navigates to the previous visible node. Does nothing if on the FIRST node
+ - <kbd>Arrow Left</kbd> - on an expanded parent node, collapses it. If on a child node, moves to its parent node.
+ - <kbd>Arrow Right</kbd> - on an expanded parent node, navigates to the first child of the node. If on a collapsed parent node, expands it.
+ - <kbd>Home</kbd> - navigates to the FIRST node
+ - <kbd>End</kbd> - navigates to the LAST visible node
+ - <kbd>Tab</kbd> - navigates to the next focusable element on the page, outside of the tree
+ - <kbd>Shift + Tab</kbd> - navigates to the previous focusable element on the page, outside of the tree
+ - <kbd>Space</kbd> - toggles selection of the current node. Marks the node as active.
+ - <kbd>Shift + Space</kbd> - toggles selection of all nodes between the active one and the one pressed Space while holding Shift if selection is enabled
+ - <kbd>Enter</kbd> - activates the focused node. If the node has link in it, open the link
+ - <kbd>*</kbd> - expands the node and all sibling nodes on the same level
+
+When selection is enabled, end-user selection of nodes is only allowed through the rendered checkbox. Since both selection types allow multiple selection, the following mouse + keyboard interactions are available
```

Comment:
```suggestion
When selection is enabled, end-user selection of nodes is only allowed through the rendered checkbox. Since both selection types allow multiple selection, the following mouse + keyboard interactions are available:
```